history.py

```diff
@@ -0,0 +1,38 @@
+from catalog import models as cmod
+
+class LastFiveMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+        #one-time configuration and initialization
+
+    def __call__(self, request):
+        #GET FROM DATABASE
+        product_ids = []
+        request.last_five = []
+        if request.session.get('something') is not None:
```

Comment:
variable name more descriptive than 'something' would be good

Reply:
That's a good point. I was not understanding what a session was, so I was getting sort of unstructured with my naming conventions. 🍦